src/components/Login/Login.css

```diff
@@ -0,0 +1,93 @@
+.button {
+  background-color: #EC6607;
+  color: white;
+  padding: 2px 6px 2px 6px;
+  border: none;
+  cursor: pointer;
+  height: 51px;
+  width: 471px;
+  font-size: 32px;
+}
+
+.buttonCenter {
+    text-align: center;
+}
+
+.inputCenter{
+    text-align: center;
+}
+
+.inputCenter label {
+    font-size: 32px;
+}
+
+input[type=email], input[type=password] {
+  width: 471px;
+  padding: 2px 2px 2px 2px;
+  height: 51px;
+}
+
+input[type=email] {
+  font-size: 28px;
```

Comment:
This and type='password' has same fontsize. It could be defined in common block above